calico/common.py

```diff
@@ -527,7 +591,7 @@ def validate_rules(profile_id, rules):
 
             action = rule.get('action')
             if (action is not None and
-                    action not in ("allow", "deny")):
+                    action not in KNOWN_ACTIONS):
```

Comment:
It'll be interpreted as "accept", I think.  Not sure it's worth policing, that behaviour feels pretty uniform to me.